### PR TITLE
new option to immediatly hide the UI if the mouse leaves interactionensource

### PR DESCRIPTION
**Pull request by oberhamsi**
see https://github.com/bitmovin/bitmovin-player-ui/issues/608

new option to immediately hide the UI when mouse leaves the UI

## Checklist (for PR submitter and reviewers)

- [ ] `CHANGELOG` entry

### CODE REVIEW

File: src/ts/components/uicontainer.ts [hunk 1]
```diff
@@ -176,7 +184,12 @@ export class UIContainer extends Container<UIContainerConfig> {
         // When a seek is going on, the seek scrub pointer may exit the UI area while still seeking, and we do not
         // hide the UI in such cases
         if (!isSeeking && !hidingPrevented()) {
-          this.uiHideTimeout.start();
+          if (this.config.hideImmediatelyOnMouseLeave) {
+            this.hideUi();
+          } else {
+            this.uiHideTimeout.start();
+          }
+
```
Comment (stonko1994) [inline src/ts/components/uicontainer.ts]: This empty line can be removed.

**Review by stonko1994 — COMMENTED** [review]
Thank you for your contribution 💪 

The changes look good and work like a charm. Please add a `CHANGELOG` entry for the new config option.

**Comment by oberhamsi** [comment]
thanks for the review

**Review by stonko1994 — APPROVED** [review]

**Comment by stonko1994** [comment]
@oberhamsi, there seem to be some merge conflicts with the base branch (most likely the Changelog). Please resolve them and then we can merge this PR.

**Pull request merged**